app/containers/Navigation.ios.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
 import React, { Component } from 'react';
-import { TabBarIOS } from 'react-native';
+import { TabBarIOS, AsyncStorage } from 'react-native';
 import { connect } from 'react-redux';
 import autobind from 'autobind-decorator';
+import FCM from 'react-native-fcm';
```

Comment:
I suspect the iOS version will have the same problem as Android, so once again this should be changed to
``import * as FCM from 'react-native-fcm';``

Otherwise the app won't start as it tries to import a function named FCM, not the whole module.